lib/core/utils/get-node-attributes.js

```diff
@@ -0,0 +1,21 @@
+/* global axe */
+
+/**
+ * Return the list of attributes of a node.
+ * @method getNodeAttributes
+ * @memberof axe.utils
+ * @param {Element} node
+ * @returns {NamedNodeMap}
+ */
+axe.utils.getNodeAttributes = function getNodeAttributes(node) {
+	// eslint-disable-next-line no-restricted-syntax
+	if (node.attributes instanceof NamedNodeMap) {
```

Comment:
Can you get this from `window.NamedNodeMap` instead? Have this not be a new global variable we require?

Reply:
Good point. Going to be a hard habit for me to break.